### PR TITLE
WT-6325 fast truncate can lead to never resolved prepared operations

### PR DESCRIPTION
**Pull request by raviprakashgiri29**

### CODE REVIEW

File: src/txn/txn.c [hunk 1]
```diff
@@ -1574,7 +1574,8 @@ __wt_txn_prepare(WT_SESSION_IMPL *session, const char *cfg[])
              * simply discarded when we find them.
              */
             for (tmp = upd->next; tmp != NULL && tmp->txnid == upd->txnid; tmp = tmp->next)
-                if (tmp->type != WT_UPDATE_RESERVE) {
+                if (tmp->type != WT_UPDATE_RESERVE &&
+                  !F_ISSET(tmp, WT_UPDATE_RESTORED_FAST_TRUNCATE)) {
```
Comment (keithbostic) [inline src/txn/txn.c:1578]: I'd suggest a comment here. Something like:
```
1574c1574,1576 
<            * simply discarded when we find them.
---            
>            * simply discarded when we find them. Also ignore updates created by instantiating
>            * fast truncation pages, they aren't linked into the transaction's modify list and so
>            * can't be considered.
```

Comment (kommiharibabu) [reply src/txn/txn.c:1578]: @raviprakashgiri29 the comment given by Keith to update the comment with his suggested text is not yet updated in the latest code. Maybe you missed to handle it?

Comment (raviprakashgiri29) [reply src/txn/txn.c:1596]: @kommiharibabu Fixed comments.

File: test/suite/test_prepare_conflict.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import wiredtiger, wttest
```
[the remark this answers is not in the document]
Comment (raviprakashgiri29) [reply test/suite/test_prepare_conflict.py:33]: Updated the file header information.

**Pull request merged**